includes/helper.php

```diff
@@ -0,0 +1,418 @@
+<?php
+
+if ( ! function_exists( 'draad_maps_icon' ) ) {
+    /**
+     * Returns an SVG icon from the icons folder
+     *
+     * @since 1.0.0
+     */
+    function draad_maps_icon( $icon )
+    {
+        $iconPath = DRAAD_MAPS_PATH . 'template-parts/icons/' . $icon . '.svg';
+
+        if ( ! file_exists( $iconPath ) ) {
+            return '';
+        }
+
+        return file_get_contents( $iconPath );
+    }
+}
+
+if ( !function_exists( 'draad_maps_get_properties' ) ) {
+    /**
+     * 
+     */
+    function draad_maps_get_properties( $data ) {
+
+        $type = '';
+        $breadcrumb = '';
+        $properties = [];
+        if ( isset( $data->features ) ) {
```

Comment:
Ik zou hier nog checken of $data wel een object is, iets als `is_object($data)`, anders kun je hier nog een exception op krijgen bij verkeerde JSON